contracts/contracts/coordination/Coordinator.sol

```diff
@@ -205,9 +207,15 @@ contract Coordinator is Ownable {
             "Node already posted aggregation"
         );
 
+        require(
+            participant.requestEncryptingKey.length == 0,
+            "Node already provided request encrypting key"
+        );
+
         // nodes commit to their aggregation result
         bytes32 aggregatedTranscriptDigest = keccak256(aggregatedTranscript);
         participant.aggregated = true;
+        participant.requestEncryptingKey = requestEncryptingKey;  // TODO validation?
```

Comment:
I had the same thought of just re-using the ferveo public blinding key as the request encrypting key in https://github.com/nucypher/nucypher/issues/3081. It would be efficient.

However, @cygnusv had some thoughts about using a different cryptographic scheme than ferveo, perhaps one more browser-compatible. The request and response encrypting keys (see #https://github.com/nucypher/nucypher/issues/3123) should probably use the same scheme so that is a consideration for the scheme we end up using. We just haven't nailed down which one to use as yet. Across the E2EE code, I've used Umbral for now as a placeholder to change later, but we should have a conversation soon about what scheme to use. If Ferveo then this PR can replace #77, if not then it would be a separate feature.